### PR TITLE
Allow where parameter for mutex unlocking

### PR DESCRIPTION
**Pull request by kravciak**
Related PR: https://github.com/os-autoinst/os-autoinst/pull/969

### CODE REVIEW

File: log_test_postgresql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+2018-06-04 15:58:00.260 CEST   [18703]LOG:  listening on Unix socket "/tmp/tmp.T7ziAC4rxp/.s.PGSQL.5432"
```
Comment (coolo) [inline log_test_postgresql]: Not sure I see the value of this file in git

File: t/13-joblocks.t [hunk 2]
```diff
@@ -179,6 +182,31 @@ sub set_token_header {
...
+# Use lock from sibling 2
+set_token_header($t->ua, 'token' . $jS2);
+
+# Check it's locked in DB
+$t->post_ok($m_prefix . '/sblock', form => {action => 'lock', where => $jS1})->status_is(200);
+$res = $schema->resultset('JobLocks')->find({owner => $jS1, name => 'sblock'});
+is($res->locked_by, $jS2, 'mutex is locked');
+
+# Check it's unlocked in DB
+$t->post_ok($m_prefix . '/sblock', form => {action => 'unlock', where => $jS1})->status_is(200);
```
Comment (coolo) [inline t/13-joblocks.t]: this is just testing the happy path. What happens if you unlock first and then try to lock? what happens if you use a different name for unlock...

Comment (kravciak) [reply t/13-joblocks.t]: Fixed, thanks

**Comment by coolo** [comment]
please ./script/tidy

**Comment by kravciak** [comment]
Done, did not realize tidy is different than on tests repo..

**Comment by coolo** [comment]
It's only for the best of reasons: historical ones :)

**Comment by okurz** [comment]
> It's only for the best of reasons: historical ones :)

You could say so. https://github.com/os-autoinst/os-autoinst/pull/662 is the best we could achieve for os-autoinst which is kinda "in sync" except for maybe the line length but not so much more. The perltidyrc from os-autoinst is the one that is also used for os-autoinst-distri-opensuse

**Pull request merged**